docs/installation/advanced/csp.md

```diff
@@ -41,7 +41,7 @@ Some CSP directives have an impact on certain rich-text editor features. Here is
 
 	**Note**: Use the more strict `img-src 'self'` if all images in the editor content are hosted from the same domain and you do **not** want to enable the {@link features/media-embed media embed} and {@link features/paste-from-word paste from Word} features.
 * `style-src 'self' 'unsafe-inline'`: `'unsafe-inline'` is necessary for:
-	* webpack's [style-loader](https://github.com/webpack-contrib/style-loader) to load the {@link framework/guides/theme-customization#styles-processing-and-bundling editor UI styles}.
+	* webpack's [style-loader](https://github.com/webpack-contrib/style-loader) to load the {@link framework/theme-customization#styles-processing-and-bundling editor UI styles}.
```

Comment:
This link doesn't work, probably missing a redirect. It should probably point here: `framework/deep-dive/ui/theme-customization.html`.